src/views/Face/D4FaceGeometry/D4FaceGeometry.tsx

```diff
@@ -64,7 +65,40 @@ const D4FaceGeometry: React.FC<Props> = ({ font, faceNum, dieFontScale, dieSize
       }
     const numbers = FACE_MAP[`${faceNum}`]
     if (!numbers) return
-    const radius = ((globalScale * d4Size) / 2) * d4RadiusScale
+
+    if(d4FontBottom){
```

Comment:
First, make sure to get the the if/else statement tabbed correctly. Second, I think that the code duplication is enough to warrant not having a separate if/else statement. What I'd do is combine your if/else statement and then do the checks in line so:
```
let rotation = d4FontBottom ? Math.PI : 0
```
and
```
if (d4FontBottom) currentGeometry.rotateZ(Math.PI)
```
Unless I'm missing something I think those might be the only two lines of code that are actually conditional.
Also, in general, I assume people working on this will be familiar with radians, so I just get the PI const directly rather than converting.